client/Components/stickyheader.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+class StickyHeader extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  showRecommendation() {
+    return (
+      <div className="recommend-percentage">
+        <span className="icon fa fa-thumbs-up"></span> {Math.trunc(100 * this.props.header.percentRec)}% <span className="recommend-text">Recommend</span>
+      </div>
+    )
+  }
+
+  showNearby() {
+    return (
+      <div className='nearby'>
+
+      </div>
+    )
+  }
+
+  render() {
+    // console.log(this.props)
+    return (
+      <div className='sticky-header sticky--hide'>
+        <div className='main'>
+          <div className='info'>
+            <h2 className='name'>{this.props.siteName}{this.props.location.verified && <span className='verified-site fa fa-check' />}</h2>
+
+            <div className='sticky-header-recs'>
+              {// checking error in accessing review service
+                !this.props.header.errorOccured
+                  ? this.showRecommendation()
+                  : <div className='empty' />
+              }
```

Comment:
Is this a different conditional rendering??? nice!! This way seems to make a little more sense than how the docs have it.

Reply:
yeah its just so that there is something there div wise but nothing is rendered on the page. not sure how the docs do it but this makes more sense for my head.